sklearn/pipeline.py

```diff
@@ -908,19 +909,29 @@ def _can_inverse_transform(self):
         return all(hasattr(t, "inverse_transform") for _, _, t in self._iter())
 
     @available_if(_can_inverse_transform)
-    def inverse_transform(self, Xt, **params):
+    def inverse_transform(self, X=None, Xt=None, **params):
```

Comment:
Some tests try to pass `X`, `y` as positional params and fallback to `X` in case of error. We need to pass other params as kwarg only
```suggestion
    def inverse_transform(self, X=None, *, Xt=None, **params):
```

Reply:
wouldn't `y` want to be explicit then? 

```python
def inverse_transform(self, X=None, y=None, *, Xt=None, **params)

Reply:
No, this function doesn't accept y at all. It's just that we force all params but ``X`` and ``y`` to be kwarg only.